packages/modelviewer.dev/examples/staging-and-camera-control.html

```diff
@@ -185,11 +185,13 @@ <h4>Event listeners can cooperate with camera-controls</h4>
 <model-viewer id="pan-demo" camera-controls src="../shared-assets/models/Astronaut.glb" alt="A 3D model of an astronaut"></model-viewer>
 <script>
 (() => {
+  
   const modelViewer = document.querySelector('#pan-demo');
+  modelViewer.addEventListener('contextmenu', event => event.preventDefault());
   const tapDistance = 2;
   let panning = false;
   let panX, panY;
-  let startX, startY;
+  let firstX, firstY;
```

Comment:
Looks like you need to pull master; these are recent changes your PR is trying to undo.